docs/pages/admin-guides/teleport-policy/database-access-controls.mdx

```diff
@@ -0,0 +1,121 @@
+
+## Prerequisites
+
+- A running Teleport Enterprise cluster v16.2.0 or later.
+- Teleport Identity and Teleport Policy enabled for your account.
+  - For self-hosted clusters, ensure that an up-to-date `license.pem` is used in the Auth Service configuration.
+- For self-hosted clusters, a running Access Graph node v1.21.3 or later.
+Check the [Teleport Policy page](./teleport-policy.mdx) for details on
+how to set up Access Graph.
+- The node running the Access Graph service must be reachable from the Teleport Auth Service.
+
+To verify that Access Graph is set up correctly for your cluster, sign in to the Teleport Web UI and navigate to the Management tab.
+If enabled, the Access Graph menu item will appear in the Permission Management section.
+
+## Example Minimal Database Access Controls Configuration
+
+This section provides an example of a Teleport configuration that corresponds to the Teleport Policy usage examples detailed later. Before deploying an updated configuration in a production environment, please ensure you've reviewed the detailed documentation on the relevant topics.
+
+### Creating a Teleport Role
+
+Create a `db_auto_user` role with the following configuration. This role:
+- Allows access to any database.
+- Enables automated user provisioning, where the database username matches the Teleport username for databases that support this feature.
+- Grants `SELECT`, `INSERT`, and `UPDATE` permissions on any objects in the `public` schema.
+
+```yaml
+kind: role
+metadata:
+  name: db_auto_user
+spec:
+  allow:
+    db_labels:
+      '*': '*'
+    db_names:
+    - '*'
+    db_permissions:
+    - match:
+        schema: public
+      permissions:
+      - SELECT
+      - INSERT
+      - UPDATE
+  options:
+    create_db_user: true
+    create_db_user_mode: keep
+version: v7
+```
+
+### Creating a New Teleport User
+
+Create a user named `alice` and assign the `db_auto_user` role to her:
+
+```bash
+> tctl users add --roles=db_auto_user alice
+```
+
+### Verifying Resource Configuration
+
+Ensure that the Postgres database resource is configured with an `admin_user` section. The admin user should be associated with a valid default database and must have privileges to create additional users. 
+
+For example:
+
+```yaml
+  - name: pg-docker
+    protocol: postgres
+    uri: localhost:5432
+    admin_user:
+      name: postgres
+      default_database: teleport
+    static_labels:
+      env: dev
+```
+
+### Further Reading
+
+For more information, refer to:
+
+- [Database Access Controls](../../enroll-resources/database-access/rbac.mdx), which covers additional topics such as custom import rules and the permission evaluation algorithm.
+- [Enrolling Self-Hosted Postgres](../../enroll-resources/database-access/enroll-self-hosted-databases/postgres-self-hosted.mdx) database.
```

Comment:
I think we usually leave these at the bottom so readers will stay on the page before clicking away at the end.